modules/phase_field/src/postprocessors/GrainTracker.C

```diff
@@ -359,6 +361,9 @@ GrainTracker::finalize()
    */
   broadcastAndUpdateGrainData();
 
+  if (_merge_grains_based_misorientaion && _t_step > 2)
+    remapGrainsMerge();
```

Comment:
The GrainTracker has a feature where we start tracking at a user-defined step (see the top of this method). I don't think it's often used, but you may want to call your method based on an offset of `_tracking_step`. Otherwise the use of that parameter may cause some oddities with the remaining logic in this method. 

Also if I may, I would suggest that we rename this method: "merge" and "remap" are both verbs so it sounds like we are remapping and merging. Some suggestions: "remapMisorientedGrains", "remapMergableGrains", "pairTwinGrains", etc. I believe the idea here is that you are trying to keep "twins" together so the method name should clearly indicate that.

Reply:
Ok, I have changed `_t_step > 2` to `_t_step > _tracking_step`, and change remapGrainsMerge to remapMisorientedGrains.